### ISSUE TITLE
[Bug]: Build Error with ironSource

### ISSUE THREAD

**Issue by gaurangbackrock** [OneSignal/OneSignal-Unity-SDK#632]
### What happened?

[CommandInvokationFailure Gradle build failed.txt](https://github.com/OneSignal/OneSignal-Unity-SDK/files/12429125/CommandInvokationFailure.Gradle.build.failed.txt)
[Picked Up Java Tool Options.txt](https://github.com/OneSignal/OneSignal-Unity-SDK/files/12429126/Picked.Up.Java.Tool.Options.txt)
[Starting a gradle demon.txt](https://github.com/OneSignal/OneSignal-Unity-SDK/files/12429127/Starting.a.gradle.demon.txt)

When I import the latest version 5.0.1, the build is not generating. I am attaching the error report in text files.
Before Importing, I could generate the build, so a Unity build error occurs when using the latest version of one signal SDK.

### Steps to reproduce?

```Markdown
In my project, I am using below SDK,
1. Firebase Analytics & Crashlytics SDK
2. Google SignIn
3. Facebook SDK
4. Ironsource Mediation (Admob, Applovin & UnityAds Adapters)
5. com.google.play.review-1.8.1 SDK
```

### What did you expect to happen?

Build should be generated when using the latest SDK.

### Unity version

2021.3.24f1

### OneSignal Unity SDK version

5.0.1

### Platform

Android

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by gaurangbackrock** [comment]
I check now by simply adding one by one all sdk into sample project and finally able to know that one adapter(UnityAds) from Ironsource is causing this error.
Please check from your side and fix this error.
Steps to reproduce error:-
1. Create sample project in Android Platform in Unity Version 2021.3.24f1.
2. Import latest OneSignal Plugin latest version 5.0.1
3. Import Ironsource Unity SDK version 7.4.0
4. Only Add Unity Ads Adapter from the Ads Mediation/Integration Manager.

Error will generate and not able to build APK.

**Comment by gaurangbackrock** [comment]
@shepherd-l 
Please can you respond me, I am waiting for your reply.
I want to know that i should wait for your reply or remove onesignal SDK as there is no solution for this error.

**Comment by shepherd-l** [comment]
@gaurangbackrock

Thanks for reporting and sorry for the late reply

I was able to reproduce your issue by following your steps and had the same error:
```
Caused by: java.lang.ClassNotFoundException: com.android.tools.lint.client.api.Vendor
```

I was able to fix the issue by enabling Custom Launcher Gradle Template in the Unity Android Build Settings and by adding the lint option `checkReleaseBuilds false` to the `launcherTemplate.gradle` :
```
android {
    ...
    lintOptions { 
	...
        checkReleaseBuilds false
    }
}
```

Let me know if this works for you

**Comment by gaurangbackrock** [comment]
@shepherd-l 
Thanks for replying me.

I tried your suggestion, but i got another error 

More than one file was found with OS independent path 'META-INF/core_release.kotlin_module'.

I tried your way but got above error and not able to fix build error.

My defualt settings are in screenshot attached.

![Screenshot 2023-08-29 at 11 30 40 AM](https://github.com/OneSignal/OneSignal-Unity-SDK/assets/139738634/4e890753-89cf-4e6a-ac26-fe1bb2ee1929)
